FirebaseStorage/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # 10.5.0
 - [added] Added Storage API to limit upload chunk size. (#10137)
+- [fixed] Run pod update or File -> Packages -> Update to latest Packages to update the GTMSessionFetcher dependency to at least version 3.1.0. This fixes an issue where it infinitely retries when FirebaseStorage returns a 500 response.
```

Comment:
This should go to next release since code freeze day is due.

Reply:
@charlotteliang The change has already happened and this is a release notes only update.  I don't think the release notes generation has begun yet.

@maneesht Please add the backquotes back around the commands and add line breaks consistent with the rest of the file

Reply:
Done.